workshops/personal_website/README.md

```diff
@@ -1,10 +1,16 @@
 # Personal Website
 
-Prophet Orpheus is here to guide you through making your own personal website. It will look something like this:
+_**We recommend going through this workshop in Google Chrome.**_
+
+Prophet Orpheus is here to guide you through making your own personal website.
```

Comment:
@paked @zachlatta What do you think of having the words "prophet orpheus" link directly to the the `hackclub/dinosaurs` repository?